src/os/xdg.zig

```diff
@@ -193,3 +193,75 @@ test parseTerminalExec {
         try testing.expectEqualSlices([*:0]const u8, actual, &.{ "a", "-e", "b", "c" });
     }
 }
+
+pub const Dirs = enum {
+    config,
+    data,
+};
```

Comment:
What do you think about somehow storing the env var key (string) and default value (string) here in `Dirs`?

Then we could reference those common values from places like:

https://github.com/ghostty-org/ghostty/blob/e2f9eb6a6f4dc2108f91293938374c0ed314dcb8/src/termio/shell_integration.zig#L479

Reply:
Excellent idea!

Reply:
One more opportunity: https://github.com/ghostty-org/ghostty/blob/e2f9eb6a6f4dc2108f91293938374c0ed314dcb8/src/termio/Exec.zig#L806-L813

Reply:
Fixed.